components/playlist/browser/playlist_service.cc

```diff
@@ -364,11 +366,7 @@ void PlaylistService::NotifyPlaylistChanged(mojom::PlaylistEvent playlist_event,
 
 void PlaylistService::NotifyMediaFilesUpdated(
     const GURL& url,
-    std::vector<mojom::PlaylistItemPtr> items) {
-  if (items.empty()) {
```

Comment:
why was this moved to the observers? That seems to make it more fragile, no?

Reply:
This check was moved here as an intermediary step (was easier to adjust tests). It's being removed everywhere in my current refactoring work, as the renderer shouldn't even send empty `base::Value::List`s.